app/grants/templates/grants/cart-vue.html

```diff
@@ -45,169 +46,426 @@
+                <div>
+                  <b-tabs @input="tabChange" v-model="tabIndex" content-class="mt-3" lazy ref="tabs">
+                    <!-- Checkout for ETH Grants -->
+                    {% include 'grants/cart/eth.html' %}
+
+                    <!-- Checkout for ZCASH Grants -->
+                    {% include 'grants/cart/qr/zcash.html' %}
+
+                    <!-- Checkout for CELO Grants -->
+                    {% include 'grants/cart/qr/celo.html' %}
+
+                    <!-- Checkout for ZIL Grants -->
+                    {% include 'grants/cart/qr/zil.html' %}
+
+                    <!-- Checkout for Harmony Grants -->
+                    {% include 'grants/cart/extension/harmony.html' %}
+
+                    <!-- Checkout for Binance Grants -->
+                    {% include 'grants/cart/extension/binance.html' %}
+
+                    <!-- Checkout for Polkadot Grants -->
+                    {% include 'grants/cart/extension/polkadot.html' %}
+
+                    <!-- Checkout for Kusama Grants -->
+                    {% include 'grants/cart/extension/kusama.html' %}
+
+                    <!-- Checkout for RSK Grants -->
+                    {% include 'grants/cart/extension/rsk.html' %}
+
+                    <!-- Checkout for Algorand Grants -->
+                    {% include 'grants/cart/extension/algorand.html' %}
+
+                    <!-- Show for non ETH tabs -->
+                    <template #tabs-end>
+                      <div role="presentation" class="nav nav-item ml-auto mt-2">
+                      <!-- d-none d-sm-block hides on screens sm or less-->
+                        <li  v-if="grantsCountByTenant.ETH == undefined" class="cursor-pointer align-self-center text-primary d-none d-md-block" @click="$refs.createCollection.fetchCollections()">
+                          <i class="far fa-heart pr-1" aria-hidden="true"></i> Save as collection
+                        </li>
+                        <li  v-if="grantsCountByTenant.ETH == undefined" class="cursor-pointer align-self-center text-primary ml-4" @click="shareCart">
+                          <i class="fa fa-share-alt pr-1" aria-hidden="true"></i> Share cart
+                        </li>
+                        <li v-if="Object.keys(grantsCountByTenant).length > 1" class="cursor-pointer align-self-center text-danger ml-4" v-b-modal.clear-cart-modal>
+                          <i class="fa fa-times pr-1" aria-hidden="true"></i> Clear all carts
+                        </li>
+                      </div>
+                    </template>
+                  </b-tabs>
+                </div>
               </div>
+              {% include 'shared/create_collection_modal.html' %}
+            </div>
+            
+            <!-- Sticky checkout for ETH -->
+            <div v-if="grantsCountByTenant.ETH !== undefined" class="bg-white border-grey border-top-2 bottom-0 px-2 py-4 position-sticky text-center w-100 d-block d-md-none">
```

Comment:
This seems very messy :/ cause now we are splitting the code in eth.html and cart-vue.html.
Can this not be kept eth.html. ?

Reply:
@thelostone-mc I was not able to solve this by keeping the code in one location. The challenge for me was to render the checkout button differently on mobile (make it sticky on mobile) and desktop, as it was designed, and I could not come up with a better solution. 
I basically needed to render it in different location, and took the approach of creating a vue component out of the button so that it is reusable, and I can render it in different places.
What is not ideal in the approach I took is that the component template was moved to cart-vue.html. Having it in a different dedicated file would have made more sense.